pkg/store/container/container.go

```diff
@@ -89,6 +95,19 @@ func (c *Container) Delete() error {
 	return c.Status.Delete()
 }
 
+// StopCh is used to pass the stop information of a container.
```

Comment:
s/pass/propagate/g

And please also change the other places.

Reply:
done.